.github/workflows/frontend-workflow.yml

```diff
@@ -70,20 +74,27 @@ jobs:
         submodules: recursive
         token: ${{ secrets.npm_token }}
 
+    - name: Setup Github as proxy Registry
+      uses: actions/setup-node@v2
+      with:
+        node-version: '${{ inputs.npm_version }}'
+        registry-url: 'https://npm.pkg.github.com/mov-ai'
+
+    - name: Setup .npmrc and install pnpm
+      if: ${{ inputs.pm == 'pnpm' }}
+      run: |
+        npm install -g pnpm
```

Comment:
I would suggest creating a new version of the agent where this runs and have this installs there.

Reply:
I can help on that

Reply:
Lets not merge this into v2. 
I will create a new branch v2.1, and that will be the target branch

Reply:
qa/npm-buildserver:v1.0.0 has it